### PR TITLE
Add ability to display time more readable

### PR DESCRIPTION
**Pull request by h4**
New attribute `friendly_time` added. It allows to display time more readable, without additional templates.

Also added option to configure sensor with common lat_lng coordinates, instead of lng_lat requested by yandex.

### CODE REVIEW

**Comment by h4** [comment]
Hi @ludeeus! Could you look at this PR please?

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
 # sensor.yandex_maps
 
 [![BuyMeCoffee][buymecoffeebedge]][buymecoffee]
-[![custom_updater](https://img.shields.io/badge/custom__updater-true-success.svg)](https://github.com/custom-components/custom_updater)
+[![custom_updater][customupdaterbedge]][customupdater]
```
Comment (ludeeus) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -32,9 +39,25 @@ key | type | description
...
+**icon** | str | `mdi:car`, can not be changed
 
 ***
 
+[cardimg]: card.png
 [exampleimg]: example.png
 [buymecoffee]: https://www.buymeacoffee.com/ludeeus
 [buymecoffeebedge]: https://camo.githubusercontent.com/cd005dca0ef55d7725912ec03a936d3a7c8de5b5/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f6275792532306d6525323061253230636f666665652d646f6e6174652d79656c6c6f772e737667
+[customupdater]: https://github.com/custom-components/custom_updater
+[customupdaterbedge]: https://img.shields.io/badge/custom__updater-true-success.svg
```
Comment (ludeeus) [inline README.md]:
```suggestion
```

File: custom_components/yandex_maps/sensor.py [hunk 3]
```diff
@@ -71,11 +80,12 @@ async def async_update(self):
                     assert resp.status == 200
                     info = await resp.json()
 
-                    self._state = info.get('direct', {}).get('time')
+                    self._state = int(info.get('direct', {}).get('time'))
```
Comment (ludeeus) [inline custom_components/yandex_maps/sensor.py]:
```suggestion
                    self._state = info.get('direct', {}).get('time')
```
States are always strings

File: custom_components/yandex_maps/sensor.py [hunk 4]
```diff
@@ -84,6 +94,14 @@ async def async_update(self):
     def is_coord(cls, data: str) -> bool:
         return bool(coords_re.fullmatch(data))
 
+    @classmethod
+    def humanize(cls, minutes):
+        _LOGGER.debug('Passed time: %s', minutes)
```
Comment (ludeeus) [inline custom_components/yandex_maps/sensor.py]:
```suggestion
    def humanize(cls, minutes):
        minutes = int(minutes)
        _LOGGER.debug('Passed time: %s', minutes)
```

**Review by ludeeus — CHANGES_REQUESTED** [review]

**Pull request closed without merging**